components/speedreader/rust/ffi/Cargo.lock

```diff
@@ -1291,18 +1291,18 @@ checksum = "1ee22067b7ccd072eeb64454b9c6e1b33b61cd0d49e895fd48676a184580e0c3"
 
 [[package]]
 name = "serde"
-version = "1.0.123"
+version = "1.0.126"
```

Comment:
This file seems to be added accidentally.

Reply:
It has been removed now.